mod/deckorate_core/set/type/answer/json.rb

```diff
@@ -27,4 +32,12 @@ def item_cards
   def atom_content?
     false
   end
+
+  def get_unit metric
```

Comment:
doesn't type/answer already have this method? It's in `Abstract::JsonldSupport`, right?